modules/nf-core/glimpse/ligate/main.nf

```diff
@@ -22,11 +22,13 @@ process GLIMPSE_LIGATE {
     def prefix = task.ext.prefix ?: "${meta.id}"
     def suffix = task.ext.suffix ?: "vcf.gz"
     """
+    printf "%s\\n" $input_list | tr -d '[],' > all_files.txt
+
     GLIMPSE_ligate \\
         $args \\
-        --input $input \\
+        --input all_files.txt \\
         --thread $task.cpus \\
-        --output ${prefix}_merged.${suffix}
+        --output ${prefix}.merged.${suffix}
```

Comment:
```suggestion
        --output ${prefix}.${suffix}
```

It's preffered to not add any hardcoded extensions. See [here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6 :)

Reply:
Okay I understand.